.github/workflows/e2e.yaml

```diff
@@ -19,9 +19,6 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - runner: sgx-ecc
-            sgx_mode: HW
-            kbc: eaa-kbc
```

Comment:
please check `payload.yaml` (and `release.yaml`) too.

Reply:
Fixed in `payload.yaml`. `release.yaml` looks good, but I'm not sure whether the  name changing `quay.io/confidential-containers/runtime-payload:enclave-cc-HW-cc-kbc` still works?